extension.js

```diff
@@ -95,6 +95,7 @@ const EasyScreenCastIndicator = GObject.registerClass({
         this.connect('button_press_event', (actor, event) =>
             this._onButtonPress(actor, event)
         );
+        this.remove_action(this._clickGesture);
```

Comment:
Fixes this error on enabling the extension: `Error: Expected an object of type ClutterAction
for argument 'action' but got type undefined`

```suggestion
        if (this._clickGesture !== null && this._clickGesture !== undefined)
            this.remove_action(this._clickGesture);
```